.github/workflows/build.yml

```diff
@@ -0,0 +1,124 @@
+---
+name: build
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
can you add "trying" and "staging" here?

Reply:
ah, but not push for master

Reply:
Huh, I thought this is the idea of this action to upload images from master to Dockerhub.